lib/cdo/aws/cloud_formation.rb

```diff
@@ -18,6 +18,8 @@ class CloudFormation
     TEMPLATE = ENV['TEMPLATE'] || raise('Stack template not provided in environment (TEMPLATE=[stack].yml.erb)')
     TEMPLATE_POLICY = TEMPLATE.split('.').tap {|s| s.first << '-policy'}.join('.')
     TEMP_BUCKET = ENV['TEMP_S3_BUCKET'] || 'cf-templates-p9nfb0gyyrpf-us-east-1'
+    # number of seconds to configure as Time To Live for DNS record
+    DNS_TTL = 60
```

Comment:
We set a constant `TTL: 60` in `cloud_formation_stack.yml.erb` in two places, that could be replaced with this defined constant (e.g.: `<%= DNS_TTL %>`):
https://github.com/code-dot-org/code-dot-org/blob/a26fcdefd89d1abbd2ae515451f98c18aaaea222/aws/cloudformation/cloud_formation_stack.yml.erb#L393
https://github.com/code-dot-org/code-dot-org/blob/a26fcdefd89d1abbd2ae515451f98c18aaaea222/aws/cloudformation/cloud_formation_stack.yml.erb#L429

Reply:
Done!